inference-engine/src/inference_engine/include/ie/gpu/gpu_context_api_ocl.hpp

```diff
@@ -243,15 +257,13 @@ static inline Blob::Ptr make_shared_blob(const TensorDesc& desc, RemoteContext::
         IE_THROW() << "Invalid remote context passed";
     }
 
-    ParamMap params = {
-        { GPU_PARAM_KEY(SHARED_MEM_TYPE), GPU_PARAM_VALUE(OCL_BUFFER) },
-        { GPU_PARAM_KEY(MEM_HANDLE), static_cast<gpu_handle_param>(buffer.get()) }
-    };
+    ParamMap params = {{GPU_PARAM_KEY(SHARED_MEM_TYPE), GPU_PARAM_VALUE(OCL_BUFFER)}, {GPU_PARAM_KEY(MEM_HANDLE), static_cast<gpu_handle_param>(buffer.get())}};
```

Comment:
it still tries to create a single line for initializer list. Can we disable touching of initializer lists?

Reply:
I don't know the clang-format option for it. For each partial case, we can disable clang-format.